src/app/layout.tsx

```diff
@@ -17,7 +18,17 @@ export default function RootLayout({
     <html lang="pt-BR">
       <body>
         <TopBar />
-        {children}
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+            height: 'calc(100vh - 160px)',
+            margin: '80px 0 0 0',
+            background: '#F3F2F2',
+          }}
+        >
+          {children}
+        </Box>
```

Comment:
On a smaller notebook screen the menu layout breaks, you need to adjust the sizes to deal with this.
![image](https://github.com/user-attachments/assets/320d09b1-db51-4859-a957-6c88a74e08ff)